app/views/incomes/YourIncomeCalculationView.scala.html

```diff
@@ -32,155 +32,146 @@
     ec: scala.concurrent.ExecutionContext)
 
 
-@printLink= {
-    <p class="flush--bottom">
-        @includes.link(
-            url=routes.YourIncomeCalculationController.printYourIncomeCalculationPage(model.empId).toString,
-            copy=Messages("tai.label.print"),
-            linkClasses=Seq("tai-print-link"))
-    </p>
-}
 
 @header = @{
     ptaHeader(
-        displayBackLink = true,
+        displayBackLink = false,
         mainHeadingText = Messages("tai.income.calculation.TaxableIncomeDetails", model.employerName),
         preHeadingText = Messages("tai.income.details.incomeReceivedToDate"),
-        additionalTopLinkContent = Some(printLink))
+        showGdsVersion = true )
 }
 
-@mainTemplate(
+@main(
     title = Messages("tai.income.calculation.TaxableIncomeDetails", model.employerName),
+    pageTitle = Some(Messages("tai.currentYearSummary.heading")),
+    backLinkContent = Messages("tai.back-link.upper"),
     authedUser = Some(user),
```

Comment:
Has the back link URL been set, or are you happy with it defaulting to the javascript.void(0)?